boards/shields/nrf7002ek_nrf7002_eb/nrf7002ek_nrf7002_eb.overlay

```diff
@@ -0,0 +1,27 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <freq.h>
+#include "nrf7002ek_nrf7002_eb_coex.overlay"
+
+&edge_connector_spi {
+	status = "okay";
+	cs-gpios = <&gpio0 11 GPIO_ACTIVE_LOW>;
+
+	nrf7002: nrf7002@0 {
+		compatible = "nordic,nrf7002-spi";
+		status = "okay";
+		reg = <0>;
+		spi-max-frequency = <DT_FREQ_M(8)>;
+
+		bucken-gpios = <&gpio0 4 (GPIO_ACTIVE_HIGH | GPIO_PULL_DOWN)>;
```

Comment:
This pin is specific to Thingy:53. I think we should use a nexus mapping (see https://github.com/zephyrproject-rtos/zephyr/pull/57636#pullrequestreview-1422103104) so that this shield definition is generic.